src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -313,7 +308,31 @@ public static void TestDecryptSimpleAes256_RsaTransferCapi()
         }
 
         [Fact]
-        [ActiveIssue(3334, PlatformID.AnyUnix)]
+        [OuterLoop(/* Leaks key on disk if interrupted */)]
+        public static void TestDecryptSimple_ExplicitSki()
```

Comment:
This test doesn't really seem to fit between SimpleAes256_RsaTransferCapi and SimpleAes256_RsaSha256.  Can you move it down to not break up a set?